src/Vlingo.Xoom.Turbo.Tests/Storage/StoreActorBuilderTest.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2012-2021 VLINGO LABS. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+using System;
+using Vlingo.Xoom.Actors;
+using Xunit;
+
+namespace Vlingo.Xoom.Turbo.Tests.Storage
+{
+  public class StoreActorBuilderTest : IDisposable
+  {
+    private readonly World _world;
+
+    public StoreActorBuilderTest()
+    {
+      _world = World.StartWithDefaults("store-actor-build-tests");
+    }
+
+    [Fact]
+    public void TestThatMySqlJournalActorIsBuilt()
+    {
+      // var journal = StoreActorBuilder.From(_world.Stage, new Model(ModelType.Command.ToString()),
```

Comment:
Skip the test, don't comment the code, unless it doesn't compile

Reply:
yes,  it doesn't compile and the production code is not implemented yet..